src/components/tasks/card/TaskTagEdit.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+import classNames from '@/components/tasks/card/card.module.scss';
+import { useTasksContext } from '@/context/tasks.context';
+
+type TaskTagPropsType = {
+    updateTaskTagLevel: (taskItemToUpdate: any, method: 'delete' | 'post') => void
+}
+
+const TaskTagEdit = ({ updateTaskTagLevel }: TaskTagPropsType) => {
+    const [newlevelValue, setNewLevelValue] = useState<string>()
+    const [newTagValue, setNewTagValue] = useState<string>()
+    const { taskLevels, taskTags } = useTasksContext()
+    
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        const tagToAdd = taskTags?.find(tag => tag.name === newTagValue)
+        const levelToAdd = taskLevels?.find(level => level.name === newlevelValue)
+        const taskItemToUpdate = {
+            itemtype: 'TASK',
```

Comment:
If we would be using 'TASK' "SKILL' in many places, would it be better to obtain this from constants?

Reply:
yes added to constants.